config/config.go

```diff
@@ -256,6 +253,60 @@ func (c *Auth) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	return nil
 }
 
+type MetricOverrides struct {
+	StringMetrics map[string][]StringMetric `yaml:"string_metrics,omitempty"`
```

Comment:
At the collector level there should only be this setting, not the whole MetricOverrides as the rest of those will only exist in the generator.